src/app/public/modules/text-editor/text-editor.component.ts

```diff
@@ -273,6 +312,11 @@ export class SkyTextEditorComponent implements AfterViewInit, ControlValueAccess
       this.adapterService.focusEditor(this.id);
     }
 
+    this.skyTextEditor = document.querySelector('div.sky-text-editor') as HTMLElement;
```

Comment:
Will this work if you have multiple text editors on one screen?

Reply:
fixed in commit: ["make work with multiple editors"](https://github.com/blackbaud/skyux-text-editor/pull/21/commits/90f79508a11ab920f7bd523ccac0de4e26cd8635)